logging/logger_test.go

```diff
@@ -149,3 +150,62 @@ func TestLogger_SetupLoggerWithJSON(t *testing.T) {
 	require.Contains(jsonOutput, "@message")
 	require.Equal(jsonOutput["@message"], "test warn msg")
 }
+
+func TestLogger_SetupLoggerWithValidLogPath(t *testing.T) {
+	t.Parallel()
+	require := require.New(t)
+
+	os.Mkdir("logs", 0744)
+	defer os.RemoveAll("./logs")
```

Comment:
Tests should generally use a temporary directory. You can use this to create a temporary directory:

```go
tmpDir := testutil.TempDir(t, t.Name())
defer os.RemoveAll(tmpDir)
```